### PR TITLE
Feature/save builtins

### PR DESCRIPTION
**Pull request by Agent49**
Support for writing/reading a subset of pythons builtin types.

### CODE REVIEW

File: tests/test_io.py [hunk 1]
```diff
@@ -369,6 +369,32 @@ def test_write_read_numpy_ndarrays(tmpdir):
...
+        any_bool=any_bool,
+        **dict_of_builtins
+    )
+
+    actual = io.read(filename)
+    assert isinstance(any_int, int)
+    assert actual['any_int'] == any_int
+    assert isinstance(any_bool, bool)
+    assert actual['any_bool'] == any_bool
+    # checks if `dict_of_builtins` is a subset of `actual`
```
Comment (f-brinkmann) [inline tests/test_io.py:394]: fancy - did not know that one!

File: pyfar/io/io.py [hunk 2]
```diff
@@ -230,12 +230,18 @@ def read(filename):
...
+            collection.pop('builtin_wrapper')
+
     return collection
 
 
 def write(filename, compress=False, **objs):
     """
-    Write any compatible pyfar object or numpy array as .far file to disk.
+    Write any compatible pyfar object or numpy array and often used builtin
+    types as .far file to disk.
```
Comment (f-brinkmann) [inline pyfar/io/io.py:244]: Are there any pyfar objects left, that we do not support? I would simplify this to:

```suggestion
    Write pyfar objects, numpy arrays, and Python built in types as .far file to disk.
```

**Review by f-brinkmann — APPROVED** [review]
Looks good to me - had only one minor suggestion for the docstring.

File: tests/test_io.py (hunk 1 above)
Comment (Agent49) [reply tests/test_io.py:394]: Yes it's nice and fancy :D

File: pyfar/io/io.py (hunk 2 above)
Comment (Agent49) [reply pyfar/io/io.py:244]: Well not all builtin types are actually supported, but those which are often used. But You're right with the pyfar objects. They are all supported at moment.

**Review by sikersten — APPROVED** [review]

**Pull request merged**